web/src/components/dashboard/anytime-actions/AnytimeActionDropdown.test.tsx

```diff
@@ -0,0 +1,308 @@
+          filename: "some-filename-license",
+        }),
+      ];
+
+      renderAnytimeActionDropdown();
+      fireEvent.click(screen.getByLabelText("Open"));
+      fireEvent.click(screen.getByText("some-license-name"));
+      fireEvent.click(screen.getByTestId("anytimeActionPrimaryButton"));
+      expect(mockPush).toHaveBeenCalledWith(`${ROUTES.licenseReinstatement}/some-url`);
+      expect(
+        mockAnalytics.event.anytime_action_button.click.go_to_anytime_action_screen
+      ).toHaveBeenCalledWith("some-filename-license");
+    });
+  });
+
+  describe("filtering and sorting", () => {
+    beforeEach(() => {
+      jest.resetAllMocks();
+      useMockRouter({});
+      useMockBusiness({});
+    });
+
+    const renderAnytimeActionDropdown = (): void => {
+      render(
+        <AnytimeActionDropdown
+          anytimeActionLinks={anytimeActionLinks}
+          anytimeActionTasks={anytimeActionTasks}
+          anytimeActionLicenseReinstatements={anytimeActionLicense}
+        />
+      );
+    };
+
+    it("sorts all actions by name", () => {
+      useMockBusiness({
+        profileData: generateProfileData({
+          industryId: "hvac-contractor",
+        }),
+        licenseData: generateLicenseData({
+          status: "EXPIRED",
+        }),
+      });
+      anytimeActionLicense = [
+        generateAnytimeActionLicenseReinstatement({
+          industryIds: ["hvac-contractor"],
+          applyToAllUsers: false,
+          name: "some-license-name",
+          urlSlug: "some-url",
+          filename: "some-filename-license",
+        }),
+        generateAnytimeActionLicenseReinstatement({
+          industryIds: ["hvac-contractor"],
+          applyToAllUsers: false,
+          name: "zzz-some-license-name",
+          urlSlug: "some-url",
+          filename: "some-filename-license-zzz",
+        }),
+      ];
+
+      renderAnytimeActionDropdown();
+      fireEvent.click(screen.getByLabelText("Open"));
+      const task = screen.getByText("some-task-name");
+      const license = screen.getByText("some-license-name");
+      const licenseLast = screen.getByText("zzz-some-license-name");
+      const link = screen.getByText("some-link-name");
+
+      expect(license.compareDocumentPosition(link)).toBe(Node.DOCUMENT_POSITION_FOLLOWING);
+      expect(link.compareDocumentPosition(task)).toBe(Node.DOCUMENT_POSITION_FOLLOWING);
+      expect(task.compareDocumentPosition(licenseLast)).toBe(Node.DOCUMENT_POSITION_FOLLOWING);
+    });
+
+    it("filters out licenses unrelated to user industry", () => {
+      useMockBusiness({
+        profileData: generateProfileData({
+          industryId: "hvac-contractor",
+        }),
+        licenseData: generateLicenseData({
+          status: "EXPIRED",
+        }),
+      });
+      anytimeActionLicense = [
+        generateAnytimeActionLicenseReinstatement({
+          industryIds: ["hvac-contractor"],
+          applyToAllUsers: false,
+          name: "some-license-name",
```

Comment:
[dust] given that we are expecting the `hvac-contractor` license reinstatement to show up and not the `cannabis` license reinstatement, thought on updating the name of the licenses to be `hvac-contractor-license` and `cannibas-license` so the expect statements are more clear?